addons/website_profile/static/src/interactions/profile_editor.js

```diff
@@ -1,77 +1,26 @@
-        ".o_forum_profile_pic_clear": { "t-on-click": this.onClearProfilePicClick },
-        ".o_forum_profile_bio_edit": {
-            "t-on-click.prevent": () => this.isEditingBio = true,
-            "t-att-class": () => ({ "d-none": this.isEditingBio }),
-        },
-        ".o_forum_profile_bio_cancel_edit": {
-            "t-on-click.prevent": () => this.isEditingBio = false,
-            "t-att-class": () => ({ "d-none": !this.isEditingBio }),
+        _root: {
+            "t-on-click.prevent": this.openDialog,
         },
-        ".o_forum_profile_bio_form": { "t-att-class": () => ({ "d-none": !this.isEditingBio }) },
-        ".o_forum_profile_bio": { "t-att-class": () => ({ "d-none": this.isEditingBio, }) },
     };
 
-    setup() {
-        this.isEditingBio = false;
-
-        this.textareaEl = this.el.querySelector("textarea.o_wysiwyg_loader");
-
-        this.options = {
-            recordInfo: {
-                context: this.services.website_page.context,
-                res_model: "res.users",
-                res_id: parseInt(this.el.querySelector("input[name=user_id]").value),
+    openDialog() {
+        this.services.dialog.add(ProfileDialog, {
+            confirm: () => {
+                browser.location.reload();
             },
-            value: this.textareaEl.getAttribute("content"),
-            resizable: true,
-            userGeneratedContent: true,
-        };
-
-        if (this.textareaEl.attributes.placeholder) {
-            this.options.placeholder = this.textareaEl.attributes.placeholder.value;
-        }
-
-        this.fileUploadEl = this.el.querySelector(".o_forum_file_upload");
-        this.avatarImgEl = this.el.querySelector(".o_wforum_avatar_img");
-    }
-
-    async willStart() {
-        await loadWysiwygFromTextarea(this, this.textareaEl, this.options);
-    }
-
-    onEditProfilePicClick() {
-        this.fileUploadEl.click();
-    }
-
-    onClearProfilePicClick() {
-        this.fileUploadEl.value = null;
-        this.avatarImgEl.src = "/web/static/img/placeholder.png";
-        const inputElement = document.createElement("input");
-        inputElement.setAttribute("name", "clear_image");
-        inputElement.setAttribute("id", "forum_clear_image");
-        inputElement.setAttribute("type", "hidden");
-        this.insert(inputElement);
-    }
-
-    onFileChange() {
-        if (!this.fileUploadEl.files.length) {
-            return;
-        }
-        const reader = new window.FileReader();
-        reader.readAsDataURL(this.fileUploadEl.files[0]);
-        this.addListener(reader, "load", (ev) => this.avatarImgEl.src = ev.target.result);
-        this.el.querySelector("#forum_clear_image")?.remove();
+            focusWebsiteDescription:
+                this.el.dataset.focusWebsiteDescription &&
+                JSON.parse(this.el.dataset.focusWebsiteDescription),
+            userId: JSON.parse(this.el.dataset.userId),
```

Comment:
JSON.parse?

Aren't those simple boolean / integer? Can't we use `this.el.dataset.focusWebsiteDescription === 'true'` and `parseInt(this.el.dataset.userId)` ?